hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestRollingRestart.java

```diff
@@ -230,6 +230,8 @@ private void waitForRSShutdownToStartAndFinish(MasterThread activeMaster,
     }
     log("Server [" + serverName + "] marked as dead, waiting for it to " +
         "finish dead processing");
+    // wait SCP be executed
+    Thread.sleep(100);
```

Comment:
We'd better not add unconditional sleeps in code. Is it possible to test whether SCP has been executed?

Reply:
Can not agree more